components/automate-cli/cmd/chef-automate/cmdUtils.go

```diff
@@ -31,6 +31,7 @@ type CmdInputs struct {
 	SkipPrintOutput          bool
 	HideSSHConnectionMessage bool
 	MutipleCmdWithArgs       map[string]string
+	InputFilesPrefix         string
```

Comment:
Move this line up, after `InputFiles               []string `
```suggestion
	InputFilesPrefix         string
```